### PR TITLE
Improve Kanban (add "quick work" note, add "backlog", rename "todo" -> "selected")

### PR DESCRIPTION
**Pull request by hiaux0**

### CODE REVIEW

**Review by JaclyNolan — COMMENTED** [review]
Understood

File: knowledge-graph/Assistant Working Guidelines/Definitions.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 #Kanban
 - [ ] Link to external resources explaining Kanban
+- You may act according to your own sense (eg. don't follow the kanban), if you deem "more efficient", but please ping/notify the Requester for a quick okay.
```
Comment (hiaux0) [inline knowledge-graph/Assistant Working Guidelines/Definitions.md:9]: important
@JaclyNolan

Comment (JaclyNolan) [reply knowledge-graph/Assistant Working Guidelines/Definitions.md:9]: Yep! Understood.

**Comment by hiaux0** [comment]
@JaclyNolan 

could you also 'approve' the PR for a visual indication? 
next time

image

**Review by JaclyNolan — COMMENTED** [review]
LGTM!

**Pull request merged**